src/tracksdata/nodes/_regionprops.py

```diff
@@ -116,18 +116,26 @@ def _axis_names(self, labels: NDArray[np.integer]) -> list[str]:
         else:
             raise ValueError(f"`labels` must be 't + 2D' or 't + 3D', got '{labels.ndim}' dimensions.")
 
-    def _init_node_attrs(self, graph: BaseGraph, axis_names: list[str]) -> None:
+    def _init_node_attrs(self, graph: BaseGraph, axis_names: list[str], labels_dtype: np.dtype | None = None) -> None:
         """
         Initialize the node attributes for the graph.
         """
         for attr_key in [DEFAULT_ATTR_KEYS.MASK, DEFAULT_ATTR_KEYS.BBOX]:
             if attr_key not in graph.node_attr_keys:
                 graph.add_node_attr_key(attr_key, None)
 
+        label_default = -1.0
+        if labels_dtype is not None and "label" in self.attr_keys() and np.issubdtype(labels_dtype, np.integer):
+            if np.issubdtype(labels_dtype, np.unsignedinteger):
+                label_default = np.array(0, dtype=labels_dtype).item()
+            else:
+                label_default = np.array(-1, dtype=labels_dtype).item()
+
```

Comment:
Hey @yfukai, just checking if I understood. Were you having problems when using an unsigned int in the `GraphArrayView`?

Reply:
Maybe the logic could be further simplified by setting the default value to `0` for labels?